pkg/forwarder/forwarder.go

```diff
@@ -274,9 +275,11 @@ func (f *DefaultForwarder) Stop() {
 func (f *DefaultForwarder) healthCheckLoop() {
 	log.Debug("Waiting for APIkey validity to be confirmed.")
 	// Wait until we confirmed we have one valid APIkey to become healthy
-	waitTicker := time.NewTicker(time.Second)
+	waitTicker := time.Tick(time.Minute * 5)
```

Comment:
Waiting 5 minutes will make container deployments too long for many clients, we should ideally target 1 minute to healthy, as our docker healthcheck is set to 2 minutes.

Is the 5 minutes set for tests or intentionnal?

Reply:
It is intentional to avoid putting too much stress on the validate endpoint (maybe it's overkill). However notice that the wait will only happend if there are no valid keys/no network after the first `validateAPIKeys` run.

Reply:
The ticker will never tick if running in docker, as we only have a 2 minutes grace period to be healthy, and will be killed if we fail to be healthy.

If we want to keep our 1 minute-to-healthy target, we should retry every 15 seconds, maybe with an exponential backoff if we don't run containerised and can stay up.